.github/workflows/tests.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
 
     - name: Download test data
       run: |
-        gdown https://drive.google.com/uc?id=18H5tzgx6LfEb7ykOg1XUuZCmhLP1Wiy8
-        unzip data.zip -d tests/
+        gdown https://drive.google.com/uc?id=1yCtgP-GTKcXtuqiuXlkmXO6Yq-nn52D0
+        unzip data_v2.zip -d tests/
```

Comment:
Table objects00 in jackson and jackson_all contains column 'confidence', which will result in an issue when 'select objects00.confidence'. So I changed the column name to 'confidence_score'